packages/client/test/sim/configs/shandong.json

```diff
@@ -0,0 +1,866 @@
+        "0x0000000000000000000000000000000000000000000000000000000000000038": "0xe71f0aa83cc32edfbefa9f4d3e0174ca85182eec9f3a09f6a6c0df6377a510d7",
+        "0x0000000000000000000000000000000000000000000000000000000000000039": "0x31206fa80a50bb6abe29085058f16212212a60eec8f049fecb92d8c8e0a84bc0",
+        "0x000000000000000000000000000000000000000000000000000000000000003a": "0x21352bfecbeddde993839f614c3dac0a3ee37543f9b412b16199dc158e23b544",
+        "0x000000000000000000000000000000000000000000000000000000000000003b": "0x619e312724bb6d7c3153ed9de791d764a366b389af13c58bf8a8d90481a46765",
+        "0x000000000000000000000000000000000000000000000000000000000000003c": "0x7cdd2986268250628d0c10e385c58c6191e6fbe05191bcc04f133f2cea72c1c4",
+        "0x000000000000000000000000000000000000000000000000000000000000003d": "0x848930bd7ba8cac54661072113fb278869e07bb8587f91392933374d017bcbe1",
+        "0x000000000000000000000000000000000000000000000000000000000000003e": "0x8869ff2c22b28cc10510d9853292803328be4fb0e80495e8bb8d271f5b889636",
+        "0x000000000000000000000000000000000000000000000000000000000000003f": "0xb5fe28e79f1b850f8658246ce9b6a1e7b49fc06db7143e8fe0b4f2b0c5523a5c",
+        "0x0000000000000000000000000000000000000000000000000000000000000040": "0x985e929f70af28d0bdd1a90a808f977f597c7c778c489e98d3bd8910d31ac0f7"
+      }
+    },
+    "0xf97e180c050e5Ab072211Ad2C213Eb5AEE4DF134": {
+      "balance": "10000000000000000000000000"
+    },
+    "0x2cA5F489CC1Fd1CEC24747B64E8dE0F4A6A850E1": {
+      "balance": "10000000000000000000000000"
+    },
+    "0x7203bd333a874D9d329050ecE393820fCD501eaA": {
+      "balance": "10000000000000000000000000"
+    },
+    "0xA51918aA40D78Ff8be939bf0E8404252875c6aDF": {
+      "balance": "10000000000000000000000000"
+    },
+    "0xAA81078e6b2121dd7A846690DFdD6b10d7658d8B": {
+      "balance": "10000000000000000000000000"
+    },
+    "0xFA2d31D8f21c1D1633E9BEB641dF77D21D63ccDd": {
+      "balance": "10000000000000000000000000"
+    },
+    "0xf751C9c6d60614226fE57D2cAD6e10C856a2ddA3": {
+      "balance": "10000000000000000000000000"
+    },
+    "0x9cD16887f6A808AEaa65D3c840f059EeA4ca1319": {
+      "balance": "10000000000000000000000000"
+    },
+    "0x2E07043584F11BFF0AC39c927665DF6c6ebaffFB": {
+      "balance": "10000000000000000000000000"
+    },
+    "0x806ce45534bb07a2CAd3a84c53611a2b3DdE316A": {
+      "balance": "10000000000000000000000000"
+    },
+    "0x97C9B168C5E14d5D369B6D88E9776E5B7b11dcC1": {
+      "balance": "10000000000000000000000000"
+    }
+  },
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "difficulty": "0x01",
+  "extraData": "",
+  "gasLimit": "0x400000",
+  "nonce": "0x1234",
+  "mixhash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "timestamp": "0"
```

Comment:
I've now pointed the Nethermind team to this genesis file for usage for an own client start, is this up to date here along the lines of the testnet restart?

Also, specific question: does this generally work with a timestamp set to `0`, so is this then picked up dynamically or something? 🤔

Or does this need to be replaced with the new timestamp `1666736008` explicitly?

Reply:
Update: comment outdated, will mark as resolved.